### PR TITLE
Create `CardChangeAccountInfo` components

### PR DESCRIPTION
**Pull request by RyanPaulGannon**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

Created:
### `BtnCaptcha.vue`
- This is a blocked out section template, I will create the issue for this to be completed

### `BtnChangeDetails.vue`
- Button to confirm changing of details

### `CardChangeAccountInfo.vue`
- Generic card for use with `username`, `email` or `password` at present

### `CardChangeAccountInfoUsername.vue`, `CardChangeAccountInfoEmail.vue`, `CardChangeAccountInfoPassword.vue`
- Added cards for `username`, `email`, `password`, maybe these could be generated by a single file?

### `index.vue`
- Added the cards to visualise upon loading of the app, can be removed after design is approved

### Related issue

- #275

### CODE REVIEW

**Review by andrewtavis — CHANGES_REQUESTED** [review]
Thanks for the work here, @RyanPaulGannon! Awesome that we are getting these components up and running 🚀 Some minor comments:

- I think we should be fine with `BtnLabeled` for the call to action change password/email/username buttons
  - For that we'd need `:cta="true"` for them. I can go through and edit them a bit post merge, but for now let's try to keep all of these in one button component so the styling is consistent even if changes happen (I have a redesign next week).
- Can we move the captcha into the `components` directory and call it `FriendlyCaptcha`?
  - We can then get started on the `Create FriendlyCaptcha.vue compoent` issue! 
  - Thanks for your offer to help make that!
- For the flex div in which we'll have the `FriendlyCaptcha` component and `BtnLabeled` we also need `items-center` to make sure that they're centered vertically
- Another thing that could be considered is moving a bit more of the styling into the `CardChangeAccountInfo` component (love the directory structure, btw! 🙌)
  - Maybe we could have `CardChangeAccountInfo` be a base class that the others call rather than the `v-if`s in `CardChangeAccountInfo`?
  - We'd then pass the text entry elements, the captcha and the CTA button into a `<slot>`?
  - I'd be happy to do this myself, but wanted to check in with you on it to get your opinion!
- Also for this one all's good, but consider adding in localization keys for the text values in the future 🌐😊

Let me know which of the changes above you'd like to send along :) :) Happy to get to the rest myself 😊

**Comment by RyanPaulGannon** [comment]
Brilliant, really appreciate the feedback & guidance! I'll take a look at the comments over the weekend/following week. I'd like to have a go, as I say I just haven't done enough front end stuff so as painful as it is I could do with pushing through and getting it done! 😅😄 Will raise the issue for the button too this weekend!

**Comment by andrewtavis** [comment]
Looking forward to the issue and the changes, @RyanPaulGannon! This stuff just takes practice, and these are already amazing first steps ✨😊 Let me know if you have any questions!

File: frontend/components/card/change-account-info/CardChangeAccountInfo.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+defineProps<{
+  header: String;
+  description: String;
+  infoType: "email" | "username" | "password";
+  ctaLabel: String;
```
Comment (rigojr) [inline frontend/components/card/change-account-info/CardChangeAccountInfo.vue:45]: Hi @RyanPaulGannon, where are `header` `description` and `ctaLabel` props being use ?

Comment (RyanPaulGannon) [reply frontend/components/card/change-account-info/CardChangeAccountInfo.vue:45]: Hey @rigojr, I've been making some adjustments recently, just haven't had the time to finish and push the update just yet :)

Comment (andrewtavis) [reply frontend/components/card/change-account-info/CardChangeAccountInfo.vue:45]: Looking forward to it, @RyanPaulGannon!

**Comment by andrewtavis** [comment]
Heyo @RyanPaulGannon 👋 Checking in on the PR here :) We're going to start more actively working on the settings pages soon, so it would be good to get this merged. Let me know if you'd like me to complete it, but also feel free to send along any final changes 😊

Hope you're well!

**Comment by andrewtavis** [comment]
Sure thing, @RyanPaulGannon! I'll merge this in and clean it up for us :) :) If you're doing some Swift atm and are looking for a project, would be great to have you over at [Scribe-iOS]! We're slowly building up the team over there (some people work on both with me), and there's lots to do that could be interesting for you to work on!

**Review by andrewtavis — APPROVED** [review]
Switching my review over to an approval given @RyanPaulGannon's capacity. I'll merge and finalize the work to be done that I mentioned above 😊

**Comment by RyanPaulGannon** [comment]
> Sure thing, @RyanPaulGannon! I'll merge this in and clean it up for us :) :) If you're doing some Swift atm and are looking for a project, would be great to have you over at [Scribe-iOS](https://github.com/scribe-org/Scribe-iOS)! We're slowly building up the team over there (some people work on both with me), and there's lots to do that could be interesting for you to work on!

I'm not doing Swift but React-Native! However, between Rust & Swift, they've both caught my eye so quite interested. I noticed the project is for language learners too, which has really peaked my interest 🤓 parce que, je voudrais parle français plus bien, so I'll certainly take a look in more depth tomorrow!

**Comment by andrewtavis** [comment]
Ya we'll be doing Rust for the Desktop client as well, so loads to learn 😊 Looking forward to the next time we can work together, @RyanPaulGannon!

**Comment by RyanPaulGannon** [comment]
Sold, looking forward too! 😬

**Pull request merged**